x11-misc/evolvotron/evolvotron-0.6.3-r1.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+inherit qmake-utils
+
+DESCRIPTION="Generative art image evolver"
+HOMEPAGE="https://sourceforge.net/projects/evolvotron/"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86 ~x86-fbsd"
+IUSE=""
+
+RDEPEND="
+	dev-libs/boost
```

Comment:
seeing that you have boost in `RDEPEND`, I assume it links to some boost lib? If so, boost needs `:=`